dockerbuild/wait_for_mongodb.sh

```diff
@@ -14,14 +14,12 @@ until mongo "$host"/foo --eval "db.getCollectionNames()" > /dev/null 2>&1; do
     >&2 echo "MongoDB is unavailable - sleeping"
     sleep 1
 done
-
 >&2 echo "MongoDB is up - executing command"
 
+
```

Comment:
Remove new blank line? You've also removed an existing blank line at 17...

Reply:
done and added the blank line at 17